Tests/PackageModelTests/SwiftSDKTests.swift

```diff
@@ -209,29 +209,29 @@ private let invalidToolsetSwiftSDKv4 = (
         },
         "schemaVersion": "4.0"
     }
-    """#
+    """# as SerializedJSON
 )
 
 private let usrBinTools = Dictionary(uniqueKeysWithValues: Toolset.KnownTool.allCases.map {
     ($0, "/usr/bin/\($0.rawValue)")
 })
 
 private let otherToolsNoRoot = (
-    path: "/tools/otherToolsNoRoot.json",
+    path: try! AbsolutePath(validating: "/tools/otherToolsNoRoot.json"),
```

Comment:
`try validating:` is not required in tests (there is an extension that does the force try)